narwhal/source_code_sharing/update_liferay_dxp_source_code_of_hotfixes.sh

```diff
@@ -282,7 +310,7 @@ function main {
 }
 
 function prepare_cache_dir {
-	install -d -m 0700 "${LIFERAY_COMMON_DOWNLOAD_CACHE_DIR}"
```

Comment:
The COMMON_CACHE_DIR should remain at its place and it should be used as it is used.
We just need a another cache dir for the local (jenkins job specific) stuff, that would conflict with other jobs.